PackageExplorer/PackageChooser/PackageDetailActionsControl.xaml.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using NuGetPe;
+using PackageExplorerViewModel;
+
+namespace PackageExplorer
+{
+    /// <summary>
+    /// Interaction logic for PackageRowDetails.xaml
+    /// </summary>
+    public partial class PackageDetailActionsControl : UserControl
+    {
```

Comment:
I'm using VS 2017 15.7 p3 and the designer works for me:
![image](https://user-images.githubusercontent.com/1427284/38771628-77dd843a-3ff4-11e8-834f-143620c41dcc.png)

I recall it working in 15.6. and 15.5 as well for WPF. the UWP designer is an issue, afaik.

Reply:
@campersau The control is showing in XAML Designer on my end as well. But I'll add a constructor that calls `InitializeComponent()` just in case.

![image](https://user-images.githubusercontent.com/1192669/38839012-9df22906-418d-11e8-961d-8fc914bb9e3e.png)

Reply:
The XAML Designer is very bugger on my end and I'm using VS2017.6.6. I had to restart the designer (https://stackoverflow.com/questions/13471097/better-way-to-restart-xaml-designer-in-visual-studio-2012) and even run `nuget restore` and `msbuild` from cmd.